moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -129,26 +150,30 @@ const ProjectNotes = ({
   let { projectId: projectIdFromParam } = useParams();
   const classes = useStyles();
   const userSessionData = getSessionDatabaseData();
+  const noteTypesIDLookup = useNoteTypeObject(
+    projectData?.moped_note_types || []
+  );
   const [noteText, setNoteText] = useState("");
   const [newNoteType, setNewNoteType] = useState(
-    isStatusEditModal ? STATUS_UPDATE_TYPE_ID : INTERNAL_NOTE_TYPE_ID
+    isStatusEditModal
+      ? noteTypesIDLookup["status_update"]
+      : noteTypesIDLookup["internal_note"]
   );
   const [editingNoteType, setEditingNoteType] = useState(null);
   const [noteAddLoading, setNoteAddLoading] = useState(false);
   const [noteAddSuccess, setNoteAddSuccess] = useState(false);
   const [isEditingNote, setIsEditingNote] = useState(false);
-  const [noteId, setNoteId] = useState(null);
-  const [displayNotes, setDisplayNotes] = useState([]);
+  const [editingNoteId, setEditingNoteId] = useState(null);
   const [filterNoteType, setFilterNoteType] = useState(
-    isStatusEditModal ? STATUS_UPDATE_TYPE_ID : 0
+    isStatusEditModal ? noteTypesIDLookup["status_update"] : 0
```

Comment:
so here maybe it could just be null instead of 0 when there is no filter being applied, feels a little more self explanatory but also i could take it or leave it :shrug: